src/containers/Tenant/Diagnostics/Overview/TableInfo/prepareTableInfo.ts

```diff
@@ -0,0 +1,96 @@
+import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
+
+import {formatObject} from '../../../../../components/InfoViewer';
```

Comment:
if the function is used more than in one component, let's move it to `helpers`.

Reply:
Currently it is not used somewhere apart from TableInfo. That’s why I placed it here

Reply:
I meant that you imported {formatObject} from '../../../../../components/InfoViewer' . If this function is used only in TableInfo, maybe you should move it to `prepareTableInfo.ts`?

Reply:
Somehow I supposed your question was about `prepareTableInfo`. So my previous comment was not about `formatObject`.

`formatObject` is also used in 3 other components: `TopicInfo`, `ChangefeedInfo` and `TopicStats`. This function applies formatter, which is return type of `createInfoFormatter`, to object. So I placed it where `createInfoFormatter`, `formatLabel`, `formatValue` functions and all connected types were declared. All formatters, which can be passed to `formatObject`, are also declared in components/InfoViewer/formatters.

Reply:
Maybe it will be more comfortable to move all this formatters to `utils/formatters`. It's just pondering, not request for immediate changes.